src/client/graphql/CreateQuoteBundle.graphql

```diff
@@ -0,0 +1,17 @@
+mutation CreateQuoteBundle($quoteCartId: ID!, $quotes: [QuoteData!]!) {
+  quoteCart_createQuoteBundle(id: $quoteCartId, input: { quoteData: $quotes }) {
+    ... on QuoteCart {
```

Comment:
Yep, we can make that optimisation but don't think it will give us much in this debugger tool, do you agree?

I think we can assume everything is fine unless we get back `CreateQuoteBundleError`.

Reply:
Yeah I agree maybe the "debugger" is not the place where those kinds of optimizations but it just feels strange and it's something I've been thinking about every time I've seen the debugger code 🤷‍♀️ But let's skip doing anything about that then.